abl/vpath/base/memory.py

```diff
@@ -77,6 +77,10 @@ def flush(self):
 
 
     def __str__(self):
```

Comment:
the `decode` transforms the bytestring into a unicode string. In python 2 we shouldn't do that, but just return the value. So, I guess, we need to redefine `__str__` for Python 2 as `__str__ = __bytes__`.

Reply:
Good catch. I guess I'll add a version check here.